config/backendConfig.ts

```diff
@@ -64,7 +66,70 @@ export const backendConfig = (): TypeInput => {
     appInfo,
     recipeList: [
       ThirdPartyEmailPasswordNode.init({
-        providers: getSocialLoginProviders()
+        providers: getSocialLoginProviders(),
+        override: {
+          apis: (originalImplementation) => {
+            return {
+              ...originalImplementation,
+
+              // override the email password sign up API
+              emailPasswordSignUpPOST: async function (input) {
+                if (originalImplementation.emailPasswordSignUpPOST === undefined) {
+                  throw Error('Should never come here')
+                }
+                // pre sign up logic goes here
+
+                const response = await originalImplementation.emailPasswordSignUpPOST(input)
+
+                if (response.status === 'OK') {
```

Comment:
Empty If statement here, any good reason to leave it in?

Reply:
> Empty If statement here, any good reason to leave it in?

I think so, this was basically copied and pasted from their docs, so it's kind of a template on which we can later implement pre/post login hooks.

Reply:
Sounds reasonable to me.